packages/nodes-base/nodes/Google/Books/GoogleBooks.node.ts

```diff
@@ -79,7 +79,7 @@ export class GoogleBooks implements INodeType {
 						'@version': [
 							1,
 						],
-					}
+					},
```

Comment:
Those fixes were applied to `master` already so simply updating your branch might solve it.

Reply:
👍 got it, i saw those errors yesterday, since they were bothering me i updated them myself